starter-code/routes/drones.js

```diff
@@ -1,15 +1,25 @@
 const express = require('express');
 
 // require the Drone model here
+const Drone = require('../models/drone');
 
 const router = express.Router();
 
 
-router.get('/drones', (req, res, next) => {
```

Comment:
Typically, I would leave this as `'/drones'` so that the `'/'` could be the home page and then drones could be the list of all drones.